public/js/views/pay-methods.jsx

```diff
@@ -6,19 +6,30 @@ import CardList from 'components/card-list';
 import { gettext } from 'utils';
 
 
-export default class ManageCards extends Component {
+export default class PaymentMethods extends Component {
 
   static propTypes = {
     closeModal: PropTypes.func.isRequired,
     paymentMethods: PropTypes.array.isRequired,
   };
 
+  renderChild() {
+    if (this.props.paymentMethods && this.props.paymentMethods.length) {
+      return <CardList cards={this.props.paymentMethods} />;
+    }
+    return (<p className="no-cards">
```

Comment:
If you call this `no-results` it will make sense when I re-use it for showing a message about not having any subscriptions :rainbow: